src/transformers/agents/prompts.py

```diff
@@ -365,7 +365,118 @@ def download_prompt(prompt_or_repo_id, agent_name, mode="run"):
 6. Don't name any new variable with the same name as a tool: for instance don't name a variable 'final_answer'.
 7. Never create any notional variables in our code, as having these in your logs might derail you from the true variables.
 8. You can use imports in your code, but only from the following list of modules: <<authorized_imports>>
-9. Don't give up! You're in charge of solving the task, not providing directions to solve it.
+9. The state persists between code executions: so if in one step you've created variables or imported modules, these will all persist.
```

Comment:
This precision helps e.g. with matplotlib plots that persist between code snippets.

Reply:
Super nice prompt!